### ISSUE TITLE
todo.txt's tasks in edit mode NOT separated by a line

### ISSUE THREAD

**Issue by mrossw** [gsantner/markor#1126]
#### General information

* v 2.4.0 F-droid
* Android 6 

#### Description
Since last 2 releases i installed, I don't get the todo.txt's tasks in edit mode separated by a line. Task ARE separated in view mode though. Any clue ?

**Comment by gsantner** [comment]
Hm, I use Markor every day and dont have any issue. Do you edit the same file on Windows as well, that may be related.

Otherwise, have you deleted app data and restarted Markor from fresh? That might help.

**Comment by mrossw** [comment]
Hi thanks for the reply. Indeed I am syncing markor files with my pc with syncthing and editing todo.txt from ubuntu too.
The lines just came back in edit mode as I marked the first item as done from the pc and it synced back to the phone.  Strange. 
I had also tried to check and uncheck highlighting option but got no chance with that. Will try delete app data as you suggest if it happens again. 
Sorry for bothering. Closing issue.
